_data/cryptocurrencies.yml

```
@@ -21,7 +21,7 @@ websites:
       - totp
       - hardware
```

```suggestion
      - u2f
```
Binance appears to support U2F based on their [documentation](https://www.binance.com/en/support/faq/360029994311).